FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -455,17 +484,27 @@ - (void)URLSession:(NSURLSession *)session
     didReceiveData:(NSData *)data {
   NSError *dataError;
   NSString *strData = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
-  strData = [strData substringFromIndex:1];
-  data = [strData dataUsingEncoding:NSUTF8StringEncoding];
-  NSDictionary *response = [NSJSONSerialization JSONObjectWithData:data
-                                                           options:NSJSONReadingMutableContainers
-                                                             error:&dataError];
-  NSString *targetTemplateVersion = _configFetch.templateVersionNumber;
-  if (dataError == nil) {
-    targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+  NSRange endRange = [strData rangeOfString:@"}"];
+  NSRange beginRange = [strData rangeOfString:@"{"];
+  if (beginRange.location != NSNotFound && endRange.location != NSNotFound) {
```

Comment:
what if beginRange is found and end is not? would that continue as expected?

Reply:
If end is not found we will just not regard it as a message. endRange is needed in order to parse it.